darts/tests/utils/test_statistics.py

```diff
@@ -136,12 +136,43 @@ def test_extract(self):
         diff = self.trend - calc_trend
         self.assertTrue(np.isclose(np.mean(diff.values() ** 2), 0.0))
 
-        # check if error is raised
+        # test MSTL method
+        calc_trend, calc_seasonality = extract_trend_and_seasonality(
+            self.ts, freq=[3, 6], method="MSTL", model=ModelMode.ADDITIVE
+        )
+        self.assertTrue(len(calc_seasonality.components) == 2)
+        diff = self.trend - calc_trend
+        # relaxed tolerance for MSTL since it will have a larger error from the
+        # extrapolation of the trend, it is still a small number but is more
+        # than STL or naive trend extraction
+        self.assertTrue(np.isclose(np.mean(diff.values() ** 2), 0.0, atol=1e-5))
+
+        # test MSTL method with single freq
+        calc_trend, calc_seasonality = extract_trend_and_seasonality(
+            self.ts, freq=6, method="MSTL", model=ModelMode.ADDITIVE
+        )
+        self.assertTrue(len(calc_seasonality.components) == 1)
+        diff = self.trend - calc_trend
+        self.assertTrue(np.isclose(np.mean(diff.values() ** 2), 0.0, atol=1e-5))
+
+        # make sure non MSTL methods fail with multiple freqs
+        with self.assertRaises(ValueError):
+            calc_trend, calc_seasonality = extract_trend_and_seasonality(
+                self.ts, freq=[1, 4, 6], method="STL", model=ModelMode.ADDITIVE
+            )
+
+        # check if error is raised when using multiplicative model
         with self.assertRaises(ValueError):
             calc_trend, _ = extract_trend_and_seasonality(
                 self.ts, freq=6, method="STL", model=ModelMode.MULTIPLICATIVE
             )
 
+        # make sure non MSTL methods fail with multiplicative model
```

Comment:
the comment does not match the test, should be removed (covered by the comment above)